game.sy

```diff
@@ -203,17 +213,39 @@ init :: fn {
     l_bind_quit("quit")
     l_bind_key("ESCAPE", "quit")
 
-    player := player_new' (0.0, 0.0)
+    player := player_new' (-0.3, 0.0)
     push' players, player
     push' entities, player
+    push' bodies, player
 
     player.puck = puck
 
-    player := player_new' (1.0, 0.0)
+    player := player_new' (0.3, 0.0)
     push' players, player
     push' entities, player
+    // push' bodies, player
 
     push' entities, puck
+    push' bodies, puck
+
+    d := 0.8
+    push' walls, collision.Wall {
+        normal: (-1.0, 0.0),
+        pos: (d, 0.0),
+    }
+    push' walls, collision.Wall {
+        normal: (1.0, 0.0),
+        pos: (-d, 0.0),
+    }
+
+    push' corners, collision.Circle {
+        rad: d,
+        pos: (0.0, 1.0 - d),
+    }
+    push' corners, collision.Circle {
+        rad: d,
+        pos: (0.0, -1.0 + d),
+    }
```

Comment:
This is placeholder stuff. It creates and arena with a rounded top and bottom and straight walls on the sides.